index.js

```diff
@@ -68,20 +90,81 @@ function Chapter (index, chapterName, fileName, content) {
   return {index: index, chapterName: chapterName, fileName: fileName, content: content}
 }
 
+Streampub.prototype._generateCover = function(async) {
```

Comment:
It's actually sort of a leftover of a few things I tried because of "write after end" thing I ran into. I will clean it up. Thanks for pointing that `async` thing out.